binding.gyp

```diff
@@ -1,4 +1,7 @@
 {
+  "variables": {
+    "fortify_source_defined": "<!(node -p \"const f='-D _FORTIFY_SOURCE='; process.env.CPPFLAGS.includes(f) || process.env.CPPFLAGS.includes(f) || process.env.CXXFLAGS.includes(f)\")",
```

Comment:
Using `node` here instead of bash-isms should ensure that it works across operating systems.